service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/transport/amqps/AmqpConnectionHandler.java

```diff
@@ -116,12 +153,11 @@ public void onConnectionBound(Event event)
                 ((TransportInternal)transport).addTransportLayer(webSocket);
             }
 
-            Sasl sasl = transport.sasl();
-            sasl.plain(this.userName, this.sasToken);
+            transport.sasl().setMechanisms("ANONYMOUS");
```

Comment:
Note that this does not mean we aren't authenticating. It just means we have a custom authorization system in place with the CBS tokens, rather than using a more AMQP standard method like "Plain"